server/adaptors/integrations/__data__/repository/aws_vpc_flow/schemas/logs_vpc-1.0.0.mapping.json

```diff
@@ -5,6 +5,9 @@
   "priority": 900,
   "data_stream": {},
   "template": {
+    "aliases": {
+      "logs-aws-vpc": {}
+    },
```

Comment:
why this alias?

Reply:
used for simple search within the dashboard